File: TKCore.Extra/Source/TKCore/Extra/Camera/Declarations.cs
using OpenTK.Mathematics;

namespace TKCore.Extra;

public partial class Camera : Movable {
    private Single BaseFieldOfView;
    private Single BaseNearPlane;
    private Single BaseFarPlane;
    private Single BaseAspectRatio;
    private Matrix4 BaseView;
    private Matrix4 BaseProjection;
}

File: TKCore.Tests/Source/TKCore/Tests/TestWindow/Declarations.vb
Imports OpenTK.Windowing.Desktop
Imports TKCore.Extra

Namespace TKCore.Tests
    Public Partial Class TestWindow : Inherits GameWindow
        Private Camera As Camera
        Private VoxelManager As VoxelManager
        Private LightManager As LightManager
        
        Private CaptureToggle As Boolean
        Private SpeedModifier As Int32
        Private Aspect As Single
    End Class
End Namespace
